tests/data_files/test-ca.opensslconf

```diff
@@ -99,3 +99,17 @@ nsCertType=server
 keyUsage = cRLSign
 subjectAltName=otherName:1.3.6.1.5.5.7.8.4;SEQ:nonprintable_hw_module_name
 nsCertType=client
+
+[directory_name_san]
+subjectAltName=dirName:dirname_sect
+
+[bad_second_directory_name_san]
+subjectAltName=dirName:dirname_sect, dirName:dirname_sect_bad
+
+[dirname_sect]
+C=UK
+O=Mbed TLS
+CN=Mbed TLS directoryName SAN
+
+[two_directorynames]
+O=MALFORM_ME
```

Comment:
Why `O=MALFORM_ME`? That doesn't seem to be part of the purpose of the `two-directorynames` test?

On the other hand, the content of `bad_second_directory_name_san` does seem related. And that section is unused. It looks like there's some kind of mistake here but I'm not sure what.

Reply:
Three files are using this extension:
- server5-two-directorynames.crt.der, which is a valid test;
- server5-directoryname-seq-malformed.crt.der, that's based on the valid file, but malforms the first directory name sequence tag;
- server5-second-directoryname-oid-malformed.crt.der, that's based on the valid file, but malforms the OID length of the second directoryName, containing "MALFORM_ME". It's just an indication that this directoryName will be malformed.

Reply:
In fact, `bad_second_directory_name_san` seems to be a leftover version of what I achieved with the malformation of the second directory name. `bad_second_directory_name_san` is unused and should be removed.